handler.go

```diff
@@ -37,12 +38,30 @@ func newHandler(config []byte) (*handler, error) {
 	if err := yaml.Unmarshal(config, &h.m); err != nil {
 		return nil, err
 	}
-	for _, e := range h.m {
-		if e.Display != "" {
-			continue
-		}
-		if strings.Contains(e.Repo, "github.com") {
+	for path, e := range h.m {
+		switch {
+		case e.Display != "":
+			// Already filled in.
+		case strings.HasPrefix(e.Repo, "https://github.com/"):
 			e.Display = fmt.Sprintf("%v %v/tree/master{/dir} %v/blob/master{/dir}/{file}#L{line}", e.Repo, e.Repo, e.Repo)
+		case strings.HasPrefix(e.Repo, "https://bitbucket.org"):
+			e.Display = fmt.Sprintf("%v %v/src/default{/dir} %v/src/default{/dir}/{file}#{file}-{line}", e.Repo, e.Repo, e.Repo)
+		}
+		if e.VCS == "" {
```

Comment:
Shouldn't we remove this if clause now? It is handled by the switch below if it cannot infer it.